### PR TITLE
show warning when pod's containers are restarting

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://github.com/openshift/origin/issues/5188

If there are three or more restarts on a single container in a pod, you get a warning in status.

@kargakis ptal

### CODE REVIEW

File: pkg/api/kubegraph/analysis/pod.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		}
+
+		for _, containerStatus := range pod.Status.ContainerStatuses {
+			if containerStatus.RestartCount >= RestartThreshold {
+				markers = append(markers, osgraph.Marker{
+					Node: podNode,
+
+					Severity: osgraph.WarningSeverity,
+					Key:      RestartingPodWarning,
+					Message: fmt.Sprintf("container %q has restarted %d times",
```
Comment (kargakis) [inline pkg/api/kubegraph/analysis/pod.go]:
```go
fmt.Sprintf("container %s in %s has restarted %d times", containerStatus.Name, podNode.ResourceString(), containerStatus.RestartCount)
```

**Comment by kargakis** [comment]
one comment, LGTM otherwise

File: pkg/api/kubegraph/analysis/pod.go (hunk 1 above)
Comment (deads2k) [inline pkg/api/kubegraph/analysis/pod.go]: > fmt.Sprintf("container %s in %s has restarted %d times", containerStatus.Name, podNode.ResourceString(), containerStatus.RestartCount)

done

**Comment by deads2k** [comment]
[test]

**Comment by liggitt** [comment]
[merge]

**Pull request merged**